services/pihole/docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       - TS_STATE_DIR=/var/lib/tailscale
       - TS_SERVE_CONFIG=/config/serve.json # Tailsacale Serve configuration to expose the web interface on your local Tailnet - remove this line if not required
       - TS_USERSPACE=false
+      - TS_HEALTHCHECK_ADDR_PORT=127.0.0.1:8080 # Enable healthcheck endpoint: "127.0.0.1:8080/healthz"
```

Comment:
I suggest switching the use of port `8080 to a random one, such as `41234`, to reduce the chance of interference.